hyperspy/exceptions.py

```diff
@@ -190,6 +190,16 @@ def __init__(self, navigation_size, expected_navigation_size):
             self.navigation_size, self.expected_navigation_size)
 
 
+class NonLinearAccessError(Exception):
```

Comment:
What's about `NonLinearAxisError` instead of `NonLinearAccessError`?

Reply:
I must have discussed too much on open access with my colleagues.